src/main/java/hudson/plugins/mercurial/MercurialSCMSource.java

```diff
@@ -238,9 +343,7 @@ protected SCMRevision retrieve(@NonNull String thingName, @NonNull TaskListener
         return null;
     }
 
-    @NonNull
-    @Override
-    protected List<Action> retrieveActions(@NonNull SCMHead head,
+    @Override protected @Nonnull List<Action> retrieveActions(@NonNull SCMHead head,
```

Comment:
aligning with annotation conventions... can remove but then I wouldn't be practicing why I preached ;-)